.github/workflows/test-workflow.yml

```diff
@@ -25,22 +25,47 @@ jobs:
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: us-west-2
 
-    - name: Render Amazon ECS task definition
-      id: render-task-definition
-      uses: aws-actions/amazon-ecs-render-task-definition@964c59d79b99c32ccbda305c9e6afb0cec542076
+    - name: Render Amazon ECS task definition with task definition file 
+      id: render-task-definition-file
+      uses: aws-actions/amazon-ecs-render-task-definition@e246ecdd89c8de5a4a8c248830566528cbed06dd
       with:
         task-definition: task-definition.json
         container-name: sample-app
         image: amazon/amazon-ecs-sample:latest
 
-    - name: Validate Amazon ECS task definition
+    - name: Validate Amazon ECS task definition file 
       env:
-        TASK_DEF_FILE: ${{ steps.render-task-definition.outputs.task-definition }}
+        TASK_DEF_FILE: ${{ steps.render-task-definition-file.outputs.task-definition }}
       run: |
         diff expected-task-definition.json $TASK_DEF_FILE
 
     - name: Register Amazon ECS task definition
       env:
-        TASK_DEF_FILE: ${{ steps.render-task-definition.outputs.task-definition }}
+        TASK_DEF_FILE: ${{ steps.render-task-definition-file.outputs.task-definition }}
       run: |
         aws ecs register-task-definition --cli-input-json file://$TASK_DEF_FILE
+
+    - name: Render Amazon ECS task definition with task definition arn (amazon resource name)
+      id: render-task-definition-arn
+      uses: aws-actions/amazon-ecs-render-task-definition@e246ecdd89c8de5a4a8c248830566528cbed06dd
+      with:
+        task-definition-arn: task-definition-arn
```

Comment:
Is the value for `task-definition-arn` being defined anywhere? I think we should either define these as variables at the top and use it here or drop the full arn here.